bee-block/src/output/basic.rs

```diff
@@ -279,6 +279,19 @@ impl BasicOutput {
 
         locked_address.unlock(unlock, inputs, context)
     }
+
+    /// Returns the address of the unlock conditions if the output is a simple deposit.
+    pub fn simple_deposit_address(&self) -> Option<&Address> {
```

Comment:
What is a simple deposit? :trollface: Maybe it should be `most_simple`

Reply:
So you'd call it `most_simple_address` ? Bit weird ?

Reply:
Also not sure, maybe `most_simple_basic_output_address` would be the clearest, but also weird because it's already a basic output
I just don't think that `simple_deposit` is clear

Reply:
Alright let me think of alternatives

Reply:
Or maybe keep the name, but change the comment so it says that the output needs to have only the address unlock condition and no features, native tokens

Reply:
Sure